==========================================================================================================================
==========================================================================================================================				
					case 2:
					clrscr();
					cout<<"\n Enter name:";
					gets(name);
					cout<<"\nEnter phone:";
					gets(phone);
					cout<<"\nEnter user id:";
					gets(userid);
					cout<<"\n Enter account id:";
					gets(acc_id);
					cout<<"\n Enter password:";
					cin>>password;
					cout<<"\n Enter balance:";
					cin>>balance;
===========================================EXCEPTIONAL HANDLING 12==========================================================
					/*
					try{
					if(strlen(name)>21 || strlen(phone)>10 || strlen(phone)<10 || strlen(userid)>21 || strlen(acc_id)>21 || password<0 || balance<0 )
					throw myException("Exception Caught .Try Again!!!!!!!!!!!!!!!!!!!!!");
					}
					catch(...)   //CAUGHT ALL EXCEPTIONS .
					{
					cout<<e.message();
					exit 0;
					}
					*/
					Current cc(password,acc_id,name,phone,balance,userid);
					login.assignc(cc);
==========================================================================================================================
==========================================================================================================================
